multiworld/envs/mujoco/sawyer_reach_torque/generate_goal_data_set.py

```diff
@@ -0,0 +1,30 @@
+from railrl.exploration_strategies.base import PolicyWrappedWithExplorationStrategy
+from railrl.exploration_strategies.ou_strategy import OUStrategy
+import numpy as np
+from railrl.policies.simple import RandomPolicy
+
+def generate_goal_data_set(env=None, num_goals=10000, goal_generation_dict=None):
```

Comment:
Let's keep this kind of code in railrl.  Basically, no code in this repo should import stuff from `railrl`.